classes/phing/filters/TailFilter.php

```diff
@@ -85,6 +85,9 @@ public function read($len = null)
         }
 
         while (($buffer = $this->in->read($len)) !== -1) {
+            // Replace "\r" and "\r\n" with "\n"
+            $buffer = str_replace(["\r\n", "\r"], "\n", $buffer);
```

Comment:
Instead of struggling with line endings, we should go by `$this->readLine()` as already implemented in `classes/phing/filters/LineContains.php` line 110. Therefor we have to rewrite the whole method.